comp/core/tagger/origindetection/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/DataDog/datadog-agent/comp/core/tagger/origindetection
```

Comment:
I think this go mod will be needed for the otel dependencies. But for the current code, is there something that requires this go mod?

Reply:
It will be required in another PR